src/pages/Setup/Stepper.tsx

```diff
@@ -10,25 +11,33 @@ export default function Stepper({ currentStep }: Props) {
   const { t } = useTranslation();
 
   return (
-    <div className="w-full">
-      <ul className="lg:flex hidden justify-center items-center gap-4">
+    <div
+      className={cn(
```

Comment:
the `cn` function is not necessary here right?

Reply:
I like to keep element-styles and `after`-styles separated